src/components/nav_bar/Navbar.test.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { TestWrapper } from '../../tests/test.utils';
-import { render } from '@testing-library/react';
+import { render, cleanup } from '@testing-library/react';
```

Comment:
good news, `cleanup` is now built-in to `render`, so this and any other occurrences can be removed 🎉